### PR TITLE
Schema

### PR DESCRIPTION
**Pull request by kensteph**
Hi there 👋!
In this PR I : 
✅  Create the Game table.
✅  Create the Author table.

### CODE REVIEW

**Review by biruk-bereka — APPROVED** [review]
Hi @kensteph, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️ Schema created correctly

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: schema/schema.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-- Create the Game table that inherits from Item
+CREATE TABLE game (
```
Comment (dfsalam) [inline schema/schema.sql:2]: The requirement for database says that "As you cannot implement inheritance in the database tables - add all properties and associations from the parent Item class as table column", I kindly recommend to add all the properties in Item to the Game class.

**Review by dfsalam — CHANGES_REQUESTED** [review]
Hi @kensteph  🙋‍♀️,
Great work so far! 💯
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾

# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights 🌞
✔️ Organized and readable code.
✔️  Great on implementing the db schema and the many-to-many relationship 👍
✔️ Descriptive PR title and summary 👍

## Required Changes ♻️
Kindly look through the comments below 👇 for required changes.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema/schema.sql (hunk 1 above)
Comment (kensteph) [reply schema/schema.sql:2]: Thank you @dfsalam for your review.

**Pull request merged**